qiskit_addon_dice_solver/dice_solver.py

```diff
@@ -248,7 +249,11 @@ def solve_fermion(
         temp_dir=temp_dir,
         clean_temp_dir=clean_temp_dir,
     )
-    return e_dice, sci_state, avg_occupancies
+    return (
+        e_dice,
+        sci_state,
+        avg_occupancies,
+    )
```

Comment:
Is this PR no longer needed? This line is just doing unnecessary reformatting, but I think the docs improvement is good.